airbyte-ci/connectors/pipelines/pipelines/consts.py

```diff
@@ -89,3 +89,26 @@ class INTERNAL_TOOL_PATHS(str, Enum):
 
 
 DAGGER_WRAP_ENV_VAR_NAME = "_DAGGER_WRAP_APPLIED"
+
+
+class CONNECTOR_TEST_STEP_ID(str, Enum):
```

Comment:
What do you think about moving this enum to `connectors/consts.py` as it's connector-specific?